extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzSchedulerImpl.java

```diff
@@ -606,15 +608,29 @@ static class InvokerJob implements Job {
         @Override
         public void execute(JobExecutionContext jobExecutionContext) throws JobExecutionException {
             if (trigger != null && trigger.invoker != null) { // could be null from previous runs
+                Context context = VertxContext.getOrCreateDuplicatedContext(vertx);
```

Comment:
We should not create a duplicated context in case of `trigger.invoker.isBlocking() && trigger.runBlockingMethodOnQuartzThread`.

Reply:
Fixed.